packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -34,6 +34,7 @@ const BoxDecoration _kDefaultRoundedBorderDecoration = BoxDecoration(
 const Color _kSelectionHighlightColor = Color(0x667FAACF);
 const Color _kInactiveTextColor = Color(0xFFC2C2C2);
 const Color _kDisabledBackground = Color(0xFFFAFAFA);
+const int _iOSHorizontalCursorOffsetPixels = -2;
```

Comment:
Add comment describing why it needs to be specified per platform, and how it's derived.